config/powermanagementsettings.cpp

```diff
@@ -46,8 +46,10 @@ namespace PowerManagementSettingsConstants
     const QString POWER_LOW_LEVEL_KEY { QL1S("powerLowLevel") };
     const QString SHOW_ICON_KEY { QL1S("showIcon") };
     const QString USE_THEME_ICONS_KEY { QL1S("useThemeIcons") };
-    const QString IDLENESS_ACTION_KEY { QL1S("idlenessAction") };
-    const QString IDLENESS_TIME_SECS_KEY { QL1S("idlenessTimeSecs") };
+    const QString IDLENESS_AC_ACTION_KEY { QL1S("idlenessACAction") };
```

Comment:
One minor note.
In this new version we completely ignore the pre-existing "legacy" configuration.
IMO we should fallback to legacy configuration for both AC/baterry if there is no new configuration defined in the config yet.

E.g. something like
```cpp
return value(IDLENESS_AC_ACTION_KEY, value(IDLENESS_LEGACY_ACTION_KEY, -1)).toInt();
```

Reply:
Ok.
I'm going to set the old values to be the actual settings of both AC/Battery watchers.